server/chat-plugins/mafia.ts

```diff
@@ -250,6 +250,8 @@ class Mafia extends Rooms.RoomGame<MafiaPlayer> {
 	hasPlurality: ID | null;
 
 	enableNL: boolean;
+	votelock: boolean;
+	votingall: boolean;
```

Comment:
nit: prefer `camelCase` for these names (`voteLock` / `votingAll`).

Reply:
will do this